biothings/web/settings/default.py

```diff
@@ -187,6 +187,7 @@
 # *****************************************************************************
 
 # Sentry project address
+SENTRY_DSN_URL = ''  # preferred
```

Comment:
the value we pass to `SENTRY_CLIENT_KEY` is the sentry dsn url, so we can just keep this setting, or make `SENTRY_CLIENT_KEY` an alias of `SENTRY_DSN_URL`.